src/components/GetStarted/Faq.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { ReactNode, useState, useCallback } from 'react'
+// @ts-ignore
+import { ButtonIcon, GU, RADIUS, useLayout, useTheme } from '@aragon/ui'
+import { Transition, animated } from 'react-spring/renderprops'
+import { fontWeight } from '../../style/font'
+import arrow from '../../assets/arrow.svg'
+
+const AnimatedDiv = animated.div
+
+type FaqProps = {
+  content: ReactNode
+  expansion: ReactNode
+}
+
+type ToggleButtonProps = {
+  onClick: () => void
+  opened: boolean
+}
+
+function Faq({ content, expansion }: FaqProps): JSX.Element {
+  const [opened, setOpened] = useState(false)
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+
+  const compactMode = layoutName === 'small'
+
+  const toggleButton = useCallback(() => {
+    setOpened((opened) => !opened)
+  }, [])
+
+  return (
+    <div
+      css={`
+        position: relative;
+        margin-bottom: ${RADIUS}px;
+      `}
+    >
+      <div
+        css={`
+          width: 100%;
+          z-index: 2;
+          height: auto;
+          padding: ${3 * GU}px ${8.5 * GU}px ${3 * GU}px ${3 * GU}px;
+          background: ${theme.surface};
+          box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.05);
+          border-radius: 8px;
+          font-weight: ${fontWeight.medium};
+          font-size: ${compactMode ? `19` : `24`}px;
+          color: ${theme.content};
+        `}
+      >
+        <ToggleButton onClick={toggleButton} opened={opened} />
+        {content}
+      </div>
+
+      <Transition
+        native
+        items={opened}
+        from={{ height: 0 }}
+        enter={{ height: 'auto' }}
+        leave={{ height: 0 }}
+      >
```

Comment:
Nice work adding these animations! I noticed that the closing feels slightly uncomfortable as there is a visible "jump". We can either take a look at this if you know what the problem might be, or remove it now and enhance it as a separate improvement during polish if we have time :)

![ezgif-2-32c85052154c](https://user-images.githubusercontent.com/11708259/96584834-b0983b00-12d6-11eb-9526-362cef5cafd5.gif)